nav2_behavior_tree/include/nav2_behavior_tree/bt_conversions.hpp

```diff
@@ -111,6 +111,26 @@ inline std::chrono::milliseconds convertFromString<std::chrono::milliseconds>(co
   return std::chrono::milliseconds(std::stoul(key.data()));
 }
 
+/**
+ * @brief Parse XML string to string array
+ * @param key XML string
+ * @return std::vector<std::string>
+ */
+template<>
+inline std::vector<std::string> convertFromString(const StringView key)
+{
+  // 7 real numbers separated by semicolons
+  auto parts = BT::splitString(key, ';');
+  std::vector<std::string> string_parts;
+  for (auto& part: parts){
+    string_parts.push_back({part.begin(), part.end()});
+  }
+
+  return string_parts;
+}
+
```

Comment:
I think there will be complaints from linting for these additional spaces. Try to run `ament_cpplint` and see if there is also other issues. 

Tip: I always run `ament_uncrustify` and `ament_cpplint` to take care of the spacing or any other formatting issues